docs/grammar/grammar.ebnf

```diff
@@ -29,15 +29,18 @@ item_decl               =   identifier, break_opt,
 field_decl              =   identifier, break_opt, ":", break_opt,
                             identifier, break_opt,
                             field_end;
+custom_union_item_decl  =   identifier, break_opt, ":", break_opt,
+                            number,     break_opt,
```

Comment:
`custom_union_item_decl` allow `0` as custom ID, it should be `number_greater_or_equal_than_zero` here.
Ref: 
https://github.com/nervosnetwork/molecule/blob/1b3bfe27e304ddd4324de807152ffc2e72028e73/tools/codegen/src/grammar.pest#L31-L35

https://github.com/nervosnetwork/molecule/blob/1b3bfe27e304ddd4324de807152ffc2e72028e73/tools/codegen/src/grammar.pest#L49-L54

Reply:
thx for the tip, improved